### PR TITLE
simplifier pass (CI): try uncommenting CI job to see if LLVM tests succeed

### PR DESCRIPTION
**Pull request by gxyd**

### CODE REVIEW

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -782,13 +782,13 @@ jobs:
...
-      #   run: |
-      #       cd integration_tests
-      #       ./run_tests.py -b llvm llvmImplicit
-      #       ./run_tests.py -b llvm llvmImplicit -f
+      - name: Test Linux LLVM 10-16
+        if: contains(matrix.llvm-version, '17') == false
+        shell: bash -e -l {0}
+        run: |
+            cd integration_tests
+            ./run_tests.py -b llvm llvmImplicit
```
Comment (Pranavchiku) [inline .github/workflows/CI.yml:790]: why do we need to try here? If it runs on local we can be sure

Comment (gxyd) [reply .github/workflows/CI.yml:790]: It works on macOS for me locally, it runs on Linux here, just to be sure.

Comment (gxyd) [reply .github/workflows/CI.yml:790]: Also, I can't test on all version of LLVM locally

Comment (Pranavchiku) [reply .github/workflows/CI.yml:790]: It usually works but yes what you did makes sense.

**Pull request merged**